v1/plugins/status/plugin_test.go

```diff
@@ -421,18 +476,45 @@ func TestPluginStartTriggerManual(t *testing.T) {
 	if !maps.Equal(result.Labels, exp.Labels) {
 		t.Fatalf("Expected: %v but got: %v", exp, result)
 	}
+}
+
+func TestPluginStartTriggerManual(t *testing.T) {
+	fixture := newTestFixture(t, nil)
+	fixture.server.ch = make(chan UpdateRequestV1)
+	defer fixture.server.stop()
+	tr := plugins.TriggerManual
+	fixture.plugin.config.Trigger = &tr
 
 	status := testStatus()
 
 	fixture.plugin.BulkUpdateBundleStatus(map[string]*bundle.Status{"test": status})
 
+	statuses := <-fixture.plugin.bulkBundleCh
+	fixture.plugin.lastBundleStatuses = statuses
+
 	// trigger the status update
 	go func() {
-		_ = fixture.plugin.Trigger(ctx)
+		_ = fixture.plugin.Trigger(context.Background())
 	}()
```

Comment:
I might not see the full picture here, but I don't understand why we need this trigger routine and can't simply call `fixture.plugin.oneShot(context.Background())` in the other routine. I think the test asserts what it's supposed to assert though, so won't hold this PR up because of this nit.